alternator/executor.cc

```diff
@@ -2474,11 +2487,56 @@ void executor::describe_single_item(const cql3::selection::selection& selection,
     }
 }
 
+uint64_t item_length(bytes_view cell, const column_definition& column) {
+    if (column.type == bytes_type) {
+        elogger.info("item_length bytes_type");
+        return 8; //
```

Comment:
I couldn't figure out what is this "8" or what's the purpose of this whole function.
It seems that for a 10MB blog it will return 8, but for a number it will print it out as a string and calculate that length. Neither is probably what DynamoDB does.